client/js/_player/components/common/clix_drag_and_drop/droppable.jsx

```diff
@@ -0,0 +1,65 @@
+import React              from 'react';
+import { DragSource }     from 'react-dnd';
+import { getEmptyImage }  from 'react-dnd-html5-backend';
+import ItemTypes          from '../draggable_item_types';
+
+const droppableSource = {
+  beginDrag(props, monitor, component) {
+    const bounds = component.node.getBoundingClientRect();
+    const body = document.getElementsByTagName('body')[0];
+    body.className = 'dragging';
+    return {
+      height: bounds.height,
+      width: bounds.width,
+      droppable: props.droppable,
+      previousZoneIndex: props.zoneIndex,
+    };
+  }
+};
+
+function collect(connect, monitor) {
+  return {
+    connectDragSource: connect.dragSource(),
+    isDragging: monitor.isDragging(),
+    connectDragPreview: connect.dragPreview(),
+  };
+}
+
+export class Droppable extends React.Component {
+  static propTypes = {
+    connectDragSource: React.PropTypes.func.isRequired,
+    connectDragPreview: React.PropTypes.func.isRequired,
+    isDragging: React.PropTypes.bool.isRequired,
+    className: React.PropTypes.string,
+    hide: React.PropTypes.bool,
+    showWhileDragging: React.PropTypes.bool,
+    style: React.PropTypes.shape({}),
+    droppable: React.PropTypes.shape({
+      text: React.PropTypes.string, // eslint-disable-line react/no-unused-prop-types
```

Comment:
Good point. We are passing droppable in the beginDrag function though, so I think I'll remove passing text.